src/main/scala/org/broadinstitute/dsde/firecloud/webservice/WorkspaceApiService.scala

```diff
@@ -85,6 +85,15 @@ trait WorkspaceApiService extends HttpService with FireCloudRequestBuilding
             }
           }
         } ~
+        pathPrefix("tags") {
+          pathPrefix("autocomplete" / Segment) { (query) =>
+            pathEnd {
+              requireUserInfo() { _ =>
+                passthrough(rawlsWorkspacesRoot + "/tags/autocomplete/%s".format(query), HttpMethods.GET)
```

Comment:
easiest way to address this is probably to create a `Uri`, let that object handle all the escaping, and pass through to `Uri.toString`.